cloud-functions/utils/requirements.txt

```diff
@@ -0,0 +1,54 @@
+antlr4-python3-runtime==4.8
+azure-core==1.25.0
+azure-identity==1.10.0
+azure-storage-blob==12.13.1
+cachetools==5.2.0
+certifi==2022.6.15
+cffi==1.15.1
+charset-normalizer==2.1.0
+click==8.1.3
+cloudevents==1.5.0
+coverage==6.4.4
+cryptography==37.0.4
+deprecation==2.1.0
+fhirpathpy==0.1.0
+Flask==2.2.2
+functions-framework==3.2.0
+google-auth==2.10.0
+gunicorn==20.1.0
+hl7==0.4.5
+idna==3.3
+isodate==0.6.1
+itsdangerous==2.1.2
+Jinja2==3.1.2
+MarkupSafe==2.1.1
+msal==1.18.0
+msal-extensions==1.0.0
+msrest==0.7.1
+numpy==1.23.2
+oauthlib==3.2.0
+packaging==21.3
+pandas==1.4.3
+phonenumbers==8.12.53
+polling==0.3.2
+portalocker==2.5.1
+pyarrow==8.0.0
+pyasn1==0.4.8
+pyasn1-modules==0.2.8
+pycountry==22.3.5
+pycparser==2.21
+pydantic==1.9.2
+PyJWT==2.4.0
+pyparsing==3.0.9
+python-dateutil==2.8.2
+pytz==2022.2.1
+PyYAML==6.0
+requests==2.28.1
+requests-oauthlib==1.3.1
+rsa==4.9
+six==1.16.0
+smartystreets-python-sdk==4.11.6
+typing_extensions==4.3.0
+urllib3==1.26.11
+watchdog==2.1.9
+Werkzeug==2.2.2
```

Comment:
Are all of these dependencies actually necessary for this module? This looks like everything that `phdi` depends on. Is this the result of `pip freeze >requirments.txt` in a virtual environment that has other stuff installed beyond what is needed here?